Firmware/Marlin_main.cpp

```diff
@@ -6531,8 +6533,11 @@ SERIAL_PROTOCOLPGM("\n\n");
     */
     case 117: {
         const char *src = strchr_pointer + 4; // "M117"
-        lcd_setstatus(*src == ' '? src + 1: src);
+        while (isspace(*src)) src++;
```

Comment:
Potential code size optimisation: `while (*src == ' ') src++;` only needs 10 bytes of flash but `while (isspace(*src)) src++;` needs 36 bytes.

Reply:
@gudnimg, thank you for reviewing this PR. Indeed, your suggestion dramatically decrease the required storage in bytes.